wunderio/next4drupal-project
# NEX-36: Update backend to drupal10

*Link to ticket:  https://wunder.atlassian.net/browse/NEX-36 *

*Changes proposed in this PR:*

![image](https://user-images.githubusercontent.com/185412/217256270-dc1eef4d-5c1b-45d2-8d5c-405516263e51.png)


Update the backend to drupal10. This includes:

* updates to modules and drupal core
* changes to lando recipe and setup
* circleci setup (not functional yet, but at least it's on par with drupal-project)
* silta setup (same as above)

I did the same changes as in this PR for the wunder drupal project: https://github.com/wunderio/drupal-project/pull/332 

I had to update our custom drush command that outputs to the user the values of the created consumer. We were using the uuid, but the site now requires the client_id (I wonder how it worked before!)

*How to test:*

Destroy local lando, and run the readme from the beginning. Destroying is safer than rebuilding this time I think, since we change a lot of low level things (like the database name etc).

To speed the testing up, you can use this big command:

```
 lando destroy -y && lando start && lando composer install && lando generate-oauth-keys && lando drush si --site-name="My great site name here" -y && lando install-recipe wunder_next_setup && lando drush wunder_next:setup-user-and-consumer && lando drush eshd -y && lando drush eshs

```

Then log into the site, create some content, menu items, etc. check that the site works!
